evm/gas-service.js

```diff
@@ -72,6 +74,8 @@ async function getGasUpdates(config, env, chain, destinationChains) {
         destinationChains = Object.keys(config.chains);
     }
 
+    const l1Chain = Object.values(config.chains)[0];
```

Comment:
it's not guaranteed to be the L1 chain, just the first valid chain. And we don't care if it's L1 or not, right?

Reply:
We don't care but kinda expecting it to be Ethereum, so the token price is more stable for conversions.

Reply:
But we can't assume it to be the case. Also, since we're using base_fee_usd, the source chain's token market price isn't really relevant anymore.

Reply:
Yeah the algorithm is not dependent for it to be ETH or L1 I will change the naming.
Good point about USD. I though we use ETH price for conversion as well, but that changes when I switched to USD